qiskit_ode/converters/pulse_to_signals.py

```diff
@@ -123,3 +123,51 @@ def get_signals(self, schedule: Schedule) -> List[PiecewiseConstant]:
                 )
 
         return list(signals.values())
+
+    @staticmethod
+    def get_awg_signals(
+        signals: List[PiecewiseConstant], if_modulation: float
+    ) -> List[PiecewiseConstant]:
+        """
+        Create signals that correspond to the output ports of an Arbitrary Waveform Generator
+        to be used with IQ mixers. For each signal in the list the number of signals is double
+        to create the I and Q components.
+
+        Args:
+            signals: A list of signals for which to create I and Q.
+            if_modulation: The intermediate frequency with which the AWG
+
+        Returns:
+            iq signals: A list of signals which is twice as long as the input list of signals.
+                For each input signal get_awg_signals returns two
+        """
+        new_signals = []
+
+        for sig in signals:
+            new_freq = sig.carrier_freq + if_modulation
```

Comment:
For this line: can you clarify what the inputs/outputs to this function are meant to represent? It looks like it's outputting equations (12) and (13), with the input signal representing the complex valued `Omega(t) = f(t)exp(iwt)`? If yes, should this be subtracting or adding `if_modulation`? (Asking as in the notes it's a minus sign.)

Reply:
Docstring has been updated 9c4eb56 . The notes subtracted the IF frequency but I updated it to an addition. For IQ mixers changing from plus to minus will typically switch from the right sideband to the left sideband.